crates/datafusion-app/Cargo.toml

```diff
@@ -16,6 +16,7 @@ datafusion-functions-parquet = { version = "0.1.0", path = "..//datafusion-funct
 datafusion-udfs-wasm = { version = "0.1.0", path = "../datafusion-udfs-wasm", features = [
   "serde",
 ], optional = true }
+datafusion-table-providers = { git = "https://github.com/jychen7/datafusion-table-providers", branch = "datafusion-45", features = ["mysql"], optional = true }
```

Comment:
Looks like `datafusion-table-providers` needs to update to DF v45.  Would you be able to make that change first and see when they plan on doing release?  I am trying to align all my DF related dependencies on v45 for the next release.

Reply:
yes, this PR is using a temp branch https://github.com/jychen7/datafusion-table-providers/pull/1, combining  https://github.com/datafusion-contrib/datafusion-table-providers/pull/249 and https://github.com/datafusion-contrib/datafusion-table-providers/pull/247.

I agree we should wait those two PRs merged, that's why this is a draft. I will post in Slack and ask if maintainer have time to help review.

Reply:
thread: https://discord.com/channels/885562378132000778/1206315256977035394/1348640250922143857